### PR TITLE
wishlist page added

### PR DESCRIPTION
**Pull request by ishant005**

### CODE REVIEW

**Review by kushalsourav — APPROVED** [review]
nice work , naming conventions is good

File: wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.main-text,.price-tag,.list{
+    text-align: center;
```
Comment (anupkgurung) [inline wishlist/wishlist.css:2]: use text align as utility class as it is used multiple times.

File: wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Wishlist</title>
+        <link rel="stylesheet" href="/style.css" />
+        <link rel="stylesheet" href="/wishlist/wishlist.css" />
+        <link rel="stylesheet" href="/productpage/product.css" />
```
Comment (anupkgurung) [inline wishlist/wishlist.html:10]: why are you using product & wishlist css if it's already included in style css ?

**Review by anupkgurung — CHANGES_REQUESTED** [review]
Please go through the comments

**Pull request merged**